calico-cloud_versioned_docs/version-18-2/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -77,13 +77,20 @@ You must download and set the executable flag each time you get a new version of
    ```
 ### Integrate the scanner into your build pipeline
 
-You can include the CLI scanner in your CI/CD pipelines (for example, Jenkins, Github actions). Ensure the following:
+You can include the Image Assurance CLI scanner in your CI/CD pipelines (for example, Jenkins, Github actions).
+
+1. Download the Image Assurance CLI scanner binary onto your CI runner.
+1. If you are running an ephemeral environment in the pipeline, include the download and update the executable steps in your pipeline to download the scanner on every execution.
+1. Create a secret containing the API token and API URL and make it available in the pipeline (for example, using a SECURE_API_TOKEN environment variable)
+1. Add the Image Assurance CLI scanner in your pipeline using the following command:
+
+   **Command**: `./tigera-scanner scan --apiurl <api_url> --token <secure_api_token> <image_name>:<tag>`
+
+   **Where**:
```

Comment:
I think we probably want to include a mention of `api_url` as well here to explain what it is.

Reply:
Added.

Reply:
Added.

Reply:
Hmm, I don't think saying it's related to "managed" clusters is correct. Did you mean to say "management"? 

Either way, I think it's more accurate to say the `api_url` specifies the path to uploading information to the Image Assurance system.

Reply:
Fixed.